packages/ui/src/components/va-accordion/VaAccordion.stories.ts

```diff
@@ -52,6 +52,38 @@ Default.play = async ({ canvasElement, step }) => {
   })
 }
 
+export const vModel = () => ({
+  components: { VaAccordion, VaCollapse },
+  data: () => ({ value: [true, false, true] }),
+  template: `
+    <va-accordion v-model="value" multiple>
+      <va-collapse
+        v-for="i in 3"
+        :key="i"
+        :header="value[i - 1] ? 'Must be opened' : 'Must be closed'"
+      >
+        Content
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+vModel.play = async ({ canvasElement, step }) => {
```

Comment:
Do we need this story at all? `Multiple` story does pretty much the same and even more.